package.json

```diff
@@ -60,6 +62,7 @@
     "gulp-babel": "^7.0.1",
     "gulp-flatten": "^0.4.0",
```

Comment:
@austin94 you can remove the `gulp` dependencies, right?

Reply:
Removed.